src/components/encoding-pane/encoding-shelf.tsx

```diff
@@ -168,7 +168,7 @@ class EncodingShelfBase extends React.PureComponent<
 
   protected onRemove() {
     const {id, handleAction} = this.props;
-    this.toggleCustomizer();
+    this.closePopup();
```

Comment:
This was needed to allow the field-customizer UI to close when clicking outside of the customizer.

Reply:
Ok but it's irrelevant for the purpose of this PR, thus making this commit not atomic. 

(No worry about it at this point since I already get it, but to simplify my review, please make your commits atomic.)